### PR TITLE
New Sequencing queues page and a new header grouping

### PR DESCRIPTION
**Pull request by aanil**
- Added a new Sequencing queues page
- Made a new header grouping for queues pages
- Renamed qpcr_pools.py to queues.py

### CODE REVIEW

**Comment by aanil** [comment]
Its up on genomics-status-stage!

File: status/queues.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                    proj_and_samples = {}
+                    conc_qpcr = ''
+                    art = Artifact(lims, uri = artifact.attrib['uri'])
+                    if method is 'MiSeq':
+                        conc_qpcr = art.udf['Concentration']
+                        is_rerun = art.udf["Rerun"]
+                    elif method is 'NovaSeq':
+                        new_art = art.parent_process.input_output_maps[0][0]
+                        i = 0
+                        while i < 4:
```
Comment (alneberg) [inline status/queues.py]: These kind of fixes needs comments... And maybe the number needs to be raised?

File: status/queues.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            pools[method] ={}
+            if queues[method].artifacts:
+                tree = ET.fromstring(queues[method].xml())
+                for artifact in tree.iter('artifact'):
+                    queue_time = artifact.find('queue-time').text
+                    container = Container(lims, uri = artifact.find('location').find('container').attrib['uri']).name
+                    attr_name = Artifact(lims, uri = artifact.attrib['uri']).name
+                    value = artifact.find('location').find('value').text
+                    proj_and_samples = {}
+                    conc_qpcr = ''
```
Comment (alneberg) [inline status/queues.py]: Add `is_rerun` here as well

**Comment by alneberg** [comment]
Maybe you want to add a label as well? 😉

**Pull request merged**